docs/dev/Demo/Slice Demo Prep/Prime-Demo-Script-Postman-Setup.md

```diff
@@ -0,0 +1,103 @@
+#### Handoff from Prime --> TOO 
+After the Prime creates a Service Item, there is a verbal handoff to the TOO, nothing technical happens
+#### Handoff TOO --> Prime
+The TOO reviews the Service Item and approves them and there is a verbal handoff to the Prime to show how a payment request is created and proof od service is uploaded
+    
+    Visual 2: Prime creates a payment request and uploads proof of service.
+#### Handoff from Prime to TOO/TIO
+After the prime uploads the proof of service, they submit the payment request ID to teh TIO and do a verbal handoff to teh TOO.
+## Getting Started:
+1. Clone the `prime_api_deliverable` repo: `git@github.com:transcom/prime_api_deliverable.git`
+2. Navigate to the `postman` folder, where you will find previous slice demo collections (e.g. 202108-slice-demo)
+3. Copy over contents from a previous collection or create a new collection from scratch.
+
+### Creating a Collection from scratch:
+Once you have cloned the `prime_api_deliverable` repo, feel free to create a new branch if someone is reviewing your script or just start working directly in master. 
+Open the file in your favorite text editor. Under the postman folder, create a new folder for our slice demo collection.
+Our naming convention for each slice demo folder is going to be: YYYYMM-slice-demo.
+
+If your slice demo is happening in September 2021, for example, you will create a folder called `202109-slice-demo`. In this folder create 4 more empty folders:
+* `payloads` - feeds the requests folder
+* `requests` - this folder contains node JS files that postman uses to build the collection.
+* `templates` - this folder contains HTML and node JS code to add visualizations that will benefit the audience during the demo.
+* `events` - this folder contains node JS files that contain the objects that feed into the templates folder for the visualizations.
+
+#### Setting up with Index.js:
+The `index.js` file is where you will store all of the elements that tell postman to create a colleciton for you.
+Outside of these folders you will add a top level file called `index.js`, where we will add our path and fs requirements requirements for node.
+
+    const fs = require('fs'); // fs stands for file system
+    const path = require('path'); // internal library for node js that allows you to use path names
+   
+Following these two requirements must create your collection constant and a path for your collection:
+
+    const Collection = require('postman-collection').Collection; // make sure to npm install so that you can use this library
+    const fileName = path.basename(__dirname + '.postman_collection.json');
+    const savePath = path.join(process.cwd(), '/collections');
+    const filePath = `${savePath}/${fileName}`;
+    
+Next build your collection. In this case we will build a collection with two requests:
+
+    var myCollection;
+    // Create a collection having two requests
+    myCollection = new Collection({
```

Comment:
I think it could be helpful to link to the postman-collection docs in this section, for more context on the syntax

Reply:
Because I'm not sure from looking at the sample code, how the Collection knows that this is a GET endpoint that expects a moveID. Is it something to do with `getMoveTaskOrderTemplate`?